apps/server/src/tasks/dto/chunkInput.ts

```diff
@@ -0,0 +1,21 @@
+import { Duration } from 'moment';
+import { Field, InputType } from '@nestjs/graphql';
+import { Interval } from '../../utils/interval.scalar';
+
+@InputType()
+export class ChunkInput {
+  @Field({
+    description: 'The time when chunk should start.',
+  })
+  start: Date;
+
+  @Field(() => Interval, {
+    description: 'Duration of the chunk.',
+  })
```

Comment:
I guess we need to add some validation here, as it can crash the system.

What do you think?

Reply:
I mean - in this pr the only place where I use the chunk [TaskChunk.tsx](https://github.com/agh-kiwis/agh-kiwis/pull/47/files#diff-eb4268bfbd88c53ed67d2ffd70023538635203d83328af24b40e93dfe182bfcc) in which i perform the following mapping:
```
export const chunkToMarkAsDoneMutationMapper = (chunk: Chunk): ChunkInput => ({
  start: chunk.start,
  duration: chunk.duration,
  isDone: !chunk.isDone,
});
```
I added those fields to chunkInput just to initialize it in the proper way, however there is no form on UI on which we could customize single chunk and change its duration - validating such form would make much sense indeed. If You prefer to validate it anyway I think that we could introduce some server side validation, but we do not validate eg. duration of the const task being craeted - mostly because of the fact that our ui forces proper data entering.